### ISSUE TITLE
Settings changes don't appear to take effect until reload

### ISSUE THREAD

**Issue by Peeja** [sawhney17/logseq-calendars-plugin#37]
### Steps to reproduce:

1. Install plugin.
2. Set up a calendar with a name (`My Calendar`) and iCalendar URL (`http://example.com/calendar.ics`).
3. Press `mod+shift+p`.
4. Type `sync` to find the calendar sync command. (Notice that it's still called "Syncing with Calendar 1").
5. Invoke it.

**Expected:** The command would be called "Syncing with My Calendar". The command would sync events from `http://example.com/calendar.ics`.
**Observed:** The command is called "Syncing with Calendar 1". The command logs a 404 error after attempting to fetch `https://calendar.google.com/calendar/ical/...`. These are the default values from the settings.

After reloading Logseq, the new settings take effect, and the Expected things above are observed. Any change to the settings appears to require a reload before it takes effect. Seems like something's maybe holding onto some existing state a little too hard?

**Comment by MatSarr** [comment]
Same for me

**Comment by kobusvdm** [comment]
Seeing excatly the same:
* These settings only taking effect after a restart:
   * Name of calendar 1
   * URL of Calendar 1

Additionally:
* I could only get it to sync with success by giving Calendar 1 in the plugin config the same exact name as in Outook
* It is not syncing event with series. Only once-of events is recognized

**Comment by l4n1skyy** [comment]
i agree! the indent under same block doesn't work even though i reload logseq
